### DISCUSSION TITLE
Acesso à dados com .NET, C#, Dapper e SQL Serve (Aula:  SqlCommand e SqlDataReader)

### DISCUSSION THREAD

**Discussion by AleLucasG** [balta-io/forum#555] [Tira Dúvidas] [answered]
Boa noite! Estou fazendo o curso  Acesso à dados com .NET, C#, Dapper e SQL Serve. 
E estou na Introdução - Aula:  SqlCommand e SqlDataReader.

No momento de conectar ao banco, o mesmo da um erro.

![image](https://github.com/balta-io/forum/assets/95228437/31dad803-2abc-4214-9c0b-c82ab256a4ab)

![image](https://github.com/balta-io/forum/assets/95228437/c48a3ed1-f93d-4d85-8c3b-1d8fd19b4b5e)

![image](https://github.com/balta-io/forum/assets/95228437/89d2d134-a543-4b6f-84ae-1f8618aeb639)

![image](https://github.com/balta-io/forum/assets/95228437/eda497d8-740a-4bcc-a250-2b0711e7fb9c)

Voltei algumas aulas para ver se fiz algo errado, mas o problemas persiste

**Comment by henriqueweege** [comment, marked as answer]
Oi, @AleLucasG ! Tudo bem?
Uma possível solução é adicionar TrustServerCertificate=True a sua connectionString.

Ficaria algo como: "Server=localhost,1433;Database=balta;User ID=sa;Password=1q2w3e4r@#$;TrustServerCertificate=True";

Espero que ajude ^^

**Comment by AleLucasG** [comment]
@henriqueweege acabei de testar a sua sugestão e deu muito certo. Muito obrigada pela ajuda! 

![image](https://github.com/balta-io/forum/assets/95228437/d70efbf6-1eef-4574-bba6-f1540a2f0c74)
